table/metadata.go

```diff
@@ -1095,7 +1095,7 @@ func (c *commonMetadata) Version() int { return c.FormatVersion }
 
 type metadataV1 struct {
 	Schema    *iceberg.Schema          `json:"schema"`
```

Comment:
```suggestion
	Schema    *iceberg.Schema          `json:"schema,omitempty"`
```

since its also omitted in v2/v3
![Screenshot 2025-03-20 at 10 16 08 AM](https://github.com/user-attachments/assets/cbb534c8-bc5f-48c6-9283-aea9f4998254)

Reply:
it's deprecated, should I still be writing it despite it being deprecated?

Reply:
ah, nvm, i see what you mean.

Reply:
i thought `omitempty` was a way to prevent serializing the field as "null".